docs/books/user-guide/using-console.adoc

```diff
@@ -17,110 +17,100 @@ specific language governing permissions and limitations
-   saslMechanisms: ANONYMOUS
+    host: 0.0.0.0
+    port: 8672
+    role: normal
+    http: true
+    httpRootDir: /usr/share/qpid-dispatch/console
+    authenticatePeer: yes
+    saslMechanisms: PLAIN
 }
 ----
+--
 
-[[the-console-files]]
-=== The Console Files
-
-The files for the console are located under the console/stand-alone
-directory in the source tree
-*  'index.html'
-*  'plugin/'
-
-Copy these files to a directory under the the html or webapps directory
-of your web server. For example, for apache tomcat the files should be
-under webapps/dispatch. Then the console is available as 'http://localhost:8080/dispatch'
-
-[[console-operation]]
-== Console Operation
-
-[[logging-in-to-a-router-network]]
-=== Logging into a Router Network
-
-image:console_login.png[image]
+. If you want to set up access to the web console from any other router in the router network, repeat this procedure for each router.
 
-Enter the address of the websockets to tcp proxy that is connected to a router in the network.
+== Accessing the web console
 
-The Autostart checkbox, when checked, will automatically log in with the previous host:port the next time you start the console.
+You can access the web console from a web browser.
 
-[[overview-page]]
-=== Overview Page
+.Procedure
 
-image:console_overview.png[image]
+. In a web browser, navigate to the web console URL.
++
+--
+The web console URL is the <host>:<port> from the `listener` that you created to serve the web console. For example: `localhost:8672`.
 
-On the overview page, aggregate information about routers, addresses, and connections is displayed.
+The {ConsoleName} opens. If you set up user name and password authentication, the *Connect* tab is displayed.
+--
 
-[[topology-page]]
-=== Topology Page
+. If necessary, log in to the web console.
++
+--
+If you set up user name and password authentication, enter your user name and password to access the web console.
 
-image:console_topology.png[image]
+The syntax for the user name is <user>@<domain>. For example: `admin@my-domain`.
+--
 
-This page displays the router network in a graphical form showing how the routers are connected and information about the individual routers and links.
+== Monitoring the router network using the web console
 
-[[list-page]]
-=== List Page
+In the web console, you use the tabs to monitor the router network.
 
-image:console_entity.png[image]
+[cols="30,70"]
+|===
+| This tab... | Provides...
 
-Displays detailed information about entities such as routers, links, addresses, memory.
+| `Overview` | Aggregate information about routers, addresses, links, connections, and logs.
 
-[[charts-page]]
-=== Charts Page
+| `Entities` | Detailed information about each AMQP management entity for each router in the router network.
 
-image:console_charts.png[image]
+| `Topology` | A graphical view of the router network. The topology shows how the routers are connected, and how messages are flowing through the network.
```

Comment:
@bhardesty Maybe we can also mention that it eventually shows clients and brokers connected through the network.